### PR TITLE
Adds example code for Document Conversion integration

### PR DESCRIPTION
**Pull request by prevosti**
The integration shown in the example converts an HTML file
to Answer Units using the Document Conversion service and
uploads the Answer Units to the Retrieve and Rank service.
After indexing, the Answer Units are searchable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,22 @@ or [Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-watson).
 ## Examples
 The [examples][examples] folder has basic and advanced examples.
 
+### Document Conversion integration with Retrieve and Rank
```
Comment (germanattanasio) [inline README.md]: Add this example below the `Document Conversion` example and make sure there is a link from the table of content.

Comment (germanattanasio) [inline README.md]: The examples listed in the readme.md are basic examples. for an advance examples add the instructions in the example file in the `/examples` folder

File: examples/document_conversion_integration.v1-experimental.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  version: 'v1',
+  url: 'https://gateway.watsonplatform.net/retrieve-and-rank/api'
+});
+
+var document_conversion = watson.document_conversion({
+  username: 'INSERT YOUR USERNAME FOR THE SERVICE HERE',
+  password: 'INSERT YOUR PASSWORD FOR THE SERVICE HERE',
+  version: 'v1-experimental'
+});
+
```
Comment (germanattanasio) [inline examples/document_conversion_integration.v1-experimental.js]: probably add a comment saying that the username and password will be different for each service and that users can't use their bluemix username and password

Comment (prevosti) [inline examples/document_conversion_integration.v1-experimental.js]: done

File: README.md (hunk 1 above)
Comment (prevosti) [inline README.md]: done

**Pull request merged**